spec/scenic/adapters/postgres/views_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+module Scenic
+  module Adapters
+    describe Postgres::Views, :db do
+      it "does not query for materialized views if not supported" do
+        connection = ActiveRecord::Base.connection
+        allow(connection).to receive(:supports_materialized_views?)
+          .and_return(false)
+        ActiveRecord::Base.connection.execute(
+          "CREATE VIEW greetings AS SELECT text 'hi' AS greeting",
+        )
+        ActiveRecord::Base.connection.execute(
+          "CREATE MATERIALIZED VIEW farewells AS SELECT text 'bye' AS text",
+        )
```

Comment:
I think means that the tests can't be run in environments where materialized views aren't supported, which would be unfortunate given the purpose of the PR. I admit I don't have a great solution to this, though.